build/pom.xml

```diff
@@ -89,7 +89,7 @@
                   <goal>single</goal>
                 </goals>
                 <configuration>
-                  <finalName>teamcity</finalName>
+                  <finalName>teamcity-conjur-plugin</finalName>

```

Comment:
What is the impact, if any, of changing this name ? Just want to make sure this doesn't break anything

Reply:
This defined the name of the resulting artifact. 'teamcity-conjur-plugin.zip' seems as a more descriptive name than the generic 'teamcity.zip'. Looking at the (bundled) TC plugins at <tc_server_url>/admin/admin.html?item=plugins I can see there's no strict convention for the names but none of the plugins is called 'teamcity.zip'.

Reply:
Would it be okay to make it match the name of the repo, i.e. `conjur-teamcity-plugin`?